scout/server/blueprints/login/templates/login/users.html

```diff
@@ -75,6 +76,15 @@
                 </div>
                 <img class="rank-shield float-end" src="{{ url_for('login.static', filename=(user.events_rank + '.svg')) }}" alt="{{ user.events_rank }}">
               </td>
+              <td>
+                {% if current_user.is_admin %}
+                <a href="{{ url_for('login.remove_user', email=user.email) }}"
```

Comment:
Hm, a little scary for sure. Perhaps an are-you-really-sure modal or a Compass like "please type this to confirm"?

Reply:
I was contemplating tne option myself. I'll add a confirm button!